trl/trainer/utils.py

```diff
@@ -670,6 +670,10 @@ def compute_accuracy(eval_pred) -> Dict[str, float]:
     predictions, labels = eval_pred
     # Here, predictions is rewards_chosen and rewards_rejected.
     # We want to see how much of the time rewards_chosen > rewards_rejected.
+    if np.array(predictions[:, 0] == predictions[:, 1], dtype=float).sum() > 0:
+        warnings.warn(
+            "Some of the predictions are equal, which is probably not what you want. In our basic setting, we set the equal predictions to be wrong."
```

Comment:
What do you think about updating the text to the following?

```suggestion
            f"There are {np.array(predictions[:, 0] == predictions[:, 1]).sum()} out of {len(predictions[:, 0])} instances where the predictions for both options are equal. As a consequence the accuracy can be misleading."
```

Reply:
It seems good!